### PR TITLE
Nc 18 add a blog section module 3.3. create a public page for blog

### PR DESCRIPTION
**Pull request by MaxMancera1**
Se crearon archivos articles, cambio en rutas, ejecutar `php artisan route:cache` y aplicar `php artisan migrate`

### CODE REVIEW

**Review by Deividisi26 — APPROVED** [review]
LGTM

File: app/Livewire/Public/Articles.php [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            if ($Articles){
+                $Articles = $Articles;
+            } else {
+                abort(404);
+            }
+
+            $layoutSet['pageTitle'] = $Articles['name'];
+            $this->pageTitle = $Articles['name'];
+        }else{
+            $Article = \App\Models\Article::orderBy('title')->get();
```
Comment (germanngc) [inline app/Livewire/Public/Articles.php]: Change to plural

```suggestion
            $Articles = \App\Models\Article::orderBy('title')->get();
```

File: app/Livewire/Public/Articles.php [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        }
+
+        return view('livewire.public.articles' . ($this->slug ? '_slug' : ''), ['articles' => $Article])
```
Comment (germanngc) [inline app/Livewire/Public/Articles.php]: Change to plural

```suggestion
        return view('livewire.public.articles' . ($this->slug ? '_slug' : ''), ['articles' => $Articles])
```

**Review by germanngc — CHANGES_REQUESTED** [review]
Apart from the code suggestions, you need to create a few more files:

### Create a articles_slug.blade.php

Use as example the bellow file:
`resources/views/livewire/public/articles_slug.blade.php`

### Add translation language files

As you can notice in multiple parts of the files you created, there are some portions of code as `__('mainmenu.articles')` this can be fixed by adding a value `articles` in the next two files:

`lang/es/mainmenu.php` and `lang/en/mainmenu.php`

For the translations that looks like `__('pages/articles.hero.h1')` you will need to create two files:

`lang/es/pages/articles.php` and `lang/en/pages/articles.php`

**Review by germanngc — APPROVED** [review]
🚀

**Review by germanngc — APPROVED** [review]
re approved 🔥

**Pull request merged**